.planning/phases/03-audit-system/03-02-SUMMARY.md

```
@@ -0,0 +1,189 @@
+);
+
+CREATE INDEX ix_audit_user_time ON api_audit_records("UserId", "TimestampUtc" DESC);
+CREATE INDEX ix_audit_tenant_time ON api_audit_records("TenantId", "TimestampUtc" DESC);
+CREATE INDEX ix_audit_timestamp ON api_audit_records("TimestampUtc");
+CREATE INDEX ix_audit_resource_time ON api_audit_records("ResourceType", "ResourceId", "TimestampUtc" DESC);
+```
+
+## Key Implementation Details
+
+**Two Audit Tables in Identity:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)